addons/payment_ogone/views/payment_ogone_templates.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    
+    <template id="inline_form">
+        <div t-attf-id="ogone_iframe_container_{{acquirer_id}}">
+            <iframe t-attf-id="oe_ogone_iframe_{{id}}"
+                    t-attf-name="oe_ogone_iframe_{{id}}"
+                    class="oe_ogone_iframe"/>
+        </div>
+    </template>
+
+    <template id="feedback" name="Payment processing page">
+        <t t-call="web.layout">
+            <t t-set="head">
+                <t t-call-assets="web.assets_common"/>
+                <t t-call-assets="web.assets_frontend"/>
+                <script type="text/javascript"
+                        src="/payment_ogone/static/src/js/payment_feedback.js"/>
+                <link rel="stylesheet"
+                      type="text/scss"
+                      href="/payment_ogone/static/src/scss/payment_form.scss"/>
```

Comment:
Because the feedback page does not depends on the assets_frontend.
The aim here was to have a feedback page inside the iframe. You don't want a button to login into your odoo instance inside your iframe when you come back.
Hopefully it will not be nedeed with the redirect...